src/closedWindows/closedCard-directive.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.closedCard')
+        .directive('closedCard', [() => {
+            return {
+                restrict: 'E',
+                templateUrl: 'closedWindows/closedCard.html',
+                scope: {
+                    cwindow: '='
```

Comment:
I'd prefer to have this variable with the full name of `closedWindow` rather than `cwindow`. This is probably personal preference though.